bin/cli

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env php
```

Comment:
IMO we don't need this file.
1. It is not used so it shouldn't be available to users
2. It is for testing purposes, so if it is really needed it should be moved into `tests` directory